main/appserver/pom.xml

```diff
@@ -133,6 +134,7 @@
         <javax.management.j2ee-api.version>1.1.1</javax.management.j2ee-api.version>
         <guava.version>13.0.1</guava.version>
 	<bean.validator.cdi.version>2.5.0-b06</bean.validator.cdi.version>
+        <soteria.version>1.0-m05</soteria.version>
```

Comment:
Please move this line to line 85 so that API and implementation versions are located together.